tests/testsuite/build.rs

```diff
@@ -81,6 +84,7 @@ fn build_with_symlink_to_path_dependency_with_build_script_in_git() {
     cargo_process("build").run()
 }
 
+#[allow(deprecated)]
 #[cargo_test]
 fn cargo_fail_with_no_stderr() {
```

Comment:
By the test name, it makes it sound like we could do `with_stderr_data("")`.  I assume there is mroe to this test than that?

Reply:
Updated to snapbox.